ast/compile.go

```diff
@@ -1468,6 +1411,21 @@ func (c *Compiler) resolveAllRefs() {
 			return false
 		})
 
+		if c.strict { // check for unused imports
```

Comment:
Consider updating the docs of this function to reflect this added responsibility.

Reply:
That has been added with the PR that this is a follow-up of, I think.